src/supervised/scaledloss.jl

```diff
@@ -1,4 +1,4 @@
-for KIND in (:MarginLoss, :DistanceLoss)
+for KIND in (:MarginLoss, :DistanceLoss, :SupervisedLoss)
```

Comment:
One note -- `ZeroOneLoss` does not work with `ScaledLoss` at the moment. How would you want to resolve this? Can't it be considered a `MarginLoss`?